Symfony/CS/Fixer/Contrib/LongArraySyntaxFixer.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
```

Comment:
I meant if we add "this" new fixer to the Symfony level, enabling the short_array fixer will cause a conflict as the long_array fixer is also enabled. Most people (my self included) use the Symfony level as base and then enable some extra fixers (like short_array). :)